texts/epn_paper.md

```diff
@@ -0,0 +1,87 @@
+We started exploring the opportunities and challenges of online conferences in late Summer 2019, well before the COVID-19 pandemic.
+We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
+After several discussions we committed to developing a community platform for online conferences, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).
+
+<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
+
+Once we started brainstorming how to organize online events, we realized that a lot of usual assumptions about scientific meetings do not apply anymore.
+Already the sizes of the events are very different: while a typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract 500 attendees.
+The duration of an online event is limited to a few hours because it needs to fit into the daily routines of participants spread across multiple hour zones, as opposed to the median duration of a conference being a full working week.
+The interactions during the session are also very different: the speaker cannot see the audience and therefore gets very little immediate feedback due to being unable to gauge the audience's reactions.
+Because mediating the interaction between the audience and the speaker becomes much more important, and because of the larger audience size, the moderators play a more important role in ensuring that everything goes smoothly.
+Finally, offline conferences allow ample time for informal discussion between the talks and during coffee breaks: this valuable aspect of scientific meetings needs to be accommodated separately.
+
+> **A typical workshop will have anywhere between 50 and 100 participants, an online conference session can easily attract 500 attendees.**
+
+<!-- #### We ran a couple of events, and saw that they work well. -->
+
+To help the community members orient themselves in a new setting, we prepared basic guidelines for all online event participants: organizers, moderators, speakers, and audience members.
+Once we felt sufficiently confident, we ran a trial workshop with our direct colleagues as both speakers and the audience.
+We used this test run to try out different formats of interaction between the audience and the speaker, check out unexpected technical problems, and collect the impressions from a broader group of colleagues.
+More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
+With this experience we were ready to hit the ground running and managed to organize a virtual session only a few days after the cancellation of the APS March Meeting.
```

Comment:
```suggestion
More importantly, however, we observed that videoconferencing has now matured to accommodate high quality live talks, where every viewer gets the front row experience.
With the gained knowledge we were ready to hit the ground running and managed to organize a virtual session only a few days after the cancellation of the APS March Meeting.
```

Reply:
Experience is broader than knowledge, and is appropriate here.